tests/Test.ADAL.NET.Integration/ManagedUsercredentialTests.cs

```diff
@@ -27,7 +57,7 @@ public void Initialize()
 
         [TestMethod]
         [Description("Positive Test for AcquireToken with an empty cache")]
-        public async Task AcquireTokenWithEmptyCachePositiveTestAsync()
+        public async Task AcquireTokenWithEmptyCache_GetsNewTokenFromService()
```

Comment:
update all tests to ensure that mocks were consumed. there is check for that in other tests that you can reference